### PR TITLE
fixed debian dependencies and instructions

### PR DESCRIPTION
**Pull request by mikhailnov**
* Moved systemd to suggested dependencies to avoid auto installing systemd on systems without systemd
* Added some other dependencies (some are probably still missing)
* Enhanced deb building instructions (dpkg-buildpackage without sudo because you said that would fix the Makefile)

### CODE REVIEW

**Comment by mikhailnov** [comment]
* .gitignore is needed because building a debian package can leave quite a lot of junk files in debian/

File: debian/control [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+Standards-Version: 3.9.6
 Homepage: https://github.com/linuxhw/hw-probe
 
 Package: hw-probe
 Architecture: all
-Depends: perl-base (>= 5), curl, hwinfo, dmidecode, pciutils, usbutils, hdparm, smartmontools, edid-decode
-Recommends: systemd, cpuid, inxi, sysstat, memtester, rfkill, xinput, i2c-tools, pnputils, mesa-utils, vainfo, vulkan-utils, upower
-Suggests: acpica-tools, hplip, numactl
+Depends: perl-base (>= 5), curl, hwinfo, dmidecode, pciutils, usbutils, hdparm, smartmontools, util-linux
+Recommends: inxi, cpuid, upower, hdparm, sysstat, memtester, edid-decode, rfkill, xinput, i2c-tools, pnputils, acpica-tools, mesa-utils, vainfo, vulkan-utils, linuxinfo, numactl, mcelog, procps, iostat, lm-sensors
```
Comment (lvc) [inline debian/control]: 1. Why we need linuxinfo?
2. sysstat already provides iostat
3. edid-decode is necessary to detect a monitor when hwinfo failed to detect it. Why it's just a recommended package?

File: debian/control [hunk 2]
```diff
@@ -1,15 +1,15 @@
 Source: hw-probe
-Section: utils
 Priority: optional
+Section: utils
 Maintainer: Mikhail Novosyolov <mikhailnov@dumalogiya.ru>
 Build-Depends: debhelper (>=9), perl-base (>= 5), make
-Standards-Version: 4.0.0
+Standards-Version: 3.9.6
```
Comment (lvc) [inline debian/control]: Why not latest 4.1.3?

Comment (mikhailnov) [reply debian/control]: deepinscreenshot_select-area_20180114173032
This spec at 3.9.6 seems to be comptible with debian >=oldstable

Comment (mikhailnov) [reply debian/control]: https://tracker.debian.org/pkg/debian-policy

**Comment by mikhailnov** [comment]
https://github.com/linuxhw/hw-probe/pull/13/commits/2eaab86e9ecfc62d32d90d932dedc4113838ffd9 fixed dependencies.

File: debian/control (hunk 1 above)
Comment (mikhailnov) [reply debian/control]: fixed by https://github.com/linuxhw/hw-probe/pull/13/commits/2eaab86e9ecfc62d32d90d932dedc4113838ffd9

**Comment by lvc** [comment]
Merged by commit f26824b182776c9d689e804e1bf0ce212e132d3d.

Thank you!

**Pull request closed without merging**